src/mam/mod_mam_riak_timed_arch_yz.erl

```diff
@@ -337,7 +337,7 @@ get_mam_pm_gdpr_data(Username, Host) ->
     Jid = jid:make({LUser, LServer, <<>>}),
     LookupParams = ?DUMMY_LOOKUP_PARAMETERS#{owner_jid := Jid},
     {ok, {_, _, Messages}} = lookup_messages([], Host, LookupParams),
-    {ok, [{Id, jid:to_binary(Jid), exml:to_binary(Packet)} || {Id, Jid, Packet} <- Messages]}.
+    {ok, [{Id, jid:to_binary(Jid2), exml:to_binary(Packet)} || {Id, Jid2, Packet} <- Messages]}.
```

Comment:
This is just taking an opportunity to improve a lint, that `Jid` was shadowing a previous variable.